### DISCUSSION TITLE
Free Iran Calendar

### DISCUSSION THREAD

**Discussion by kiumad** [WeAreMahsaAmini/Core#99] [Ideas]
Hello friends
A few days ago I worked on a small project and I want to give an explanation about it

### General view
In 43 years ago, activities and fateful epics were done for the freedom of Iran by the people
But most of them have been forgotten, because they are not recorded anywhere
The calendar is responsible for recording these events, but Iran's current calendar has no place for these important events

### Project goal
By launching this service, we can complete it over time so that in the future and after the freedom of our beloved Iran, we can appreciate the influential people of this path.

### Live Demo and Github Repository
live demo : https://freeirancalendar.vercel.app/
repository : https://github.com/kiumad/freeIran/tree/master/app

**Comment by mshariat121** [comment]
I love it.

**Reply by kiumad** [reply]
thank you 🙏🏻

**Reply by sinahnyazdi** [reply]
That's a great Idea.
https://www.instagram.com/barkhi_az_honarmandan/  
This page has been doing this for a long time and maybe a connection can be established between the two projects.

**Comment by kaveh6** [comment]
that's a great idea 👍

**Comment by allahyar** [comment]
Bravo 👍, I think it is better if there is a timeline.
